src/main/java/view/InputView.java

```diff
@@ -0,0 +1,34 @@
+package view;
+
+import coordinate.Point;
+import coordinate.Points;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class InputView {
+    Scanner scanner = new Scanner(System.in);
+
+    public Points inputPoint(){
+        System.out.println("좌표를 입력하세요.");
+        String s = scanner.next();
+
+        Points points = new Points();
+        String[] strings = s.split("-");
+        for(String str: strings){
+            List<Integer> numbers = new ArrayList<>();
+            List<String> nums;
+            nums = Arrays.stream(str.split("\\(|,|\\)")).filter(t -> !t.isEmpty()).collect(Collectors.toList());
```

Comment:
이렇게 한 번에 선언할 수 있어보입니다. 

```suggestion
            List<String> nums = Arrays.stream(str.split("\\(|,|\\)")).filter(t -> !t.isEmpty()).collect(Collectors.toList());
```